app/javascript/components/mentoring/discussion/IterationsList.tsx

```diff
@@ -21,6 +23,23 @@ export const IterationsList = ({
           selected={current === iteration}
         />
       ))}
+
+      <button
+        type="button"
+        aria-label="Go to previous iteration"
+        onClick={() => onClick(iterations[currentIndex - 1])}
+        disabled={iterations[0] === current}
+      >
+        Previous
+      </button>
+      <button
+        type="button"
+        aria-label="Go to next iteration"
+        onClick={() => onClick(iterations[currentIndex + 1])}
+        disabled={iterations[iterations.length - 1] === current}
+      >
+        Next
+      </button>
```

Comment:
This is fine *but* this is technically pagination and thus it _could_ be implemented as:

```html
<a
  href={urlThatGoesThere}
  onClick={handlerThatPreventsDefaultAndGoesThere}
  rel="next"
```

That way, users can mid or right click and store the url _to the iteration_.

Also, you probably want to _focus_ the new/linked iteration after "opening" it.

Reply:
@SleeplessByte I'd like to do this as well. What's a good approach to update the URL link once we click on the button?